views/screen.jade

```diff
@@ -1,7 +1,7 @@
 extends layout
 
 block scripts
-  script( src='/js/libs/engine.io/engine.io.js' )
+  script( src='/socket.io/socket.io.js' )
```

Comment:
because socket.io has its own javascript client. The same as on backend, we must replace engine-client by socket-client